app/scripts/services/globalPromotionService.ts

```diff
@@ -83,6 +83,19 @@ export class GlobalPromotionService {
     this.globalPromotionsCache.put(cacheKey, [...cachedData, newPromotion]);
   }
 
+  getPromotionsForConference(
+    conferenceMinistryId: string,
+    conferenceMinistryActivityId: string,
+  ): GlobalPromotion[] {
+    const cacheKey = this.getCacheKey(
+      conferenceMinistryId,
+      conferenceMinistryActivityId,
+    );
+    const cachedData =
+      this.globalPromotionsCache.get<GlobalPromotion[]>(cacheKey);
+    return cachedData || [];
+  }
+
   hasPromotionsForConference(
```

Comment:
Could `hasPromotionsForConference` now call `getPromotionsForConference` instead of duplicating logic?